capstone/src/main/webapp/script.js

```diff
@@ -259,6 +259,14 @@ function drawBarChart(toxicityData) {
 
 /** Shows the avaiable attributes given a language selected on text analyzer page */
 function showAvailableAttributes() {
+  $('.checkbox-menu').on('change', "input[type='checkbox']", function() {
+    $(this).closest('li').toggleClass('active', this.checked);
+  });
+
+  $(document).on('click', '.allow-focus', function(e) {
```

Comment:
Can you add a short comment explaining this?  IIUC it's because you don't want the parent HTML element to also get a `click` callback but it would be good to document this